### PR TITLE
Mark delete-frame as interactive

### PR DESCRIPTION
**Pull request by agraven**
Fixes users not being able to close frames, solving issue #1134

### CODE REVIEW

File: rust_src/src/frames.rs [hunk 1]
```diff
@@ -418,7 +418,12 @@ pub fn frame_bottom_divider_width(frame: LispFrameOrSelected) -> i32 {
 /// This function runs `delete-frame-functions' before actually
 /// deleting the frame, unless the frame is a tooltip.
 /// The functions are run with one argument, the frame to be deleted.
-#[lisp_fn(min = "0", name = "delete-frame", c_name = "delete_frame")]
+#[lisp_fn(
+    intspec = "",
```
Comment (shaleh) [inline rust_src/src/frames.rs]: We usually add the `intspec` to the end like with `DEFUN` in C. But the order is not really important.

**Comment by Wilfred** [comment]
@agraven I've added you as a collaborator, so you can also review/merge other people's PRs if you wish :)

cc @brotzeit

**Comment by brotzeit** [comment]
@Wilfred Thanks.

@agraven Unfortunately it's not possible to review changes to the wiki, so just push you're changes.

**Comment by shaleh** [comment]
You can make a branch. There is no formal support but we could read it and give you feedback.

**Pull request merged**